db/mycroft/geography_schema/delete_duplicate_cities.sql

```diff
@@ -1 +1,5 @@
-DELETE FROM geography.city WHERE id IN %(city_ids)s
+DELETE FROM
+    geography.city
+WHERE
+    id IN %(city_ids)s
+    and (population is null or population != %(max_population)s)
```

Comment:
Just to make sure I'm clear about this, it equates to:
- If there's no population it's probably not a city
- and keep the version of the city with the highest population?

Reply:
Oops.  What is this doing here?  Must have forgotten to slip it in to the last pull request.  You are correct, though.